### PR TITLE
Ad ament about good code

### PR DESCRIPTION
**Pull request by BenHSmith13**
This mostly just fixes lint errors. I was looking for code that was potentially dangerous to ADA, I think we were good on this one.

### CODE REVIEW

**Comment by dittonjs** [comment]
looks good to me

File: client/js/scorm/components/scorm/courses_list.jsx [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
+import React from 'react';
+import Courses from './course';
 
+export default function courseList(props) {
   return (
     <div>
       <ul className="c-list">
-        {items}
+        {
+          props.list.map((item, key) => (
```
Comment (BenHSmith13) [inline client/js/scorm/components/scorm/courses_list.jsx]: make lodash

File: client/js/scorm/middleware/redirect.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
-import { Constants as PackageConstants }   from "../actions/scorm";
-import { DONE }    from "../../constants/wrapper";
+import { Constants as PackageConstants } from '../actions/scorm';
 
-const Redirect = store => next => action => {
-  switch(action.type){
+const Redirect = store => next => (action) => { // eslint-disable-line no-unused-vars
```
Comment (BenHSmith13) [inline client/js/scorm/middleware/redirect.js]: change to _store

Comment (BenHSmith13) [inline client/js/scorm/middleware/redirect.js]: Refactor middleware at another time. Fine for now

**Pull request merged**